dumper/app/src/main/java/com/google/edwmigration/dumper/application/dumper/connector/ZonedIntervalIterable.java

```diff
@@ -170,15 +49,9 @@ private ZonedIntervalIterable(
         "Start date %s must precede end date %s",
         queryLogStart,
         queryLogEnd);
-    this.start = queryLogStart.truncatedTo(unit);
-    this.end = queryLogEnd.truncatedTo(unit);
 
-    if (!start.equals(queryLogStart)) {
-      LOG.warn("Start time has been truncated to {}", start);
-    }
-    if (!end.equals(queryLogEnd)) {
-      LOG.warn("End time has been truncated to {}", end);
-    }
+    this.start = queryLogStart;
```

Comment:
I would suggest decoupling the `ZonedIntervalIterable` class from query logs concept. These two should not be related and one should be able to reason about `ZonedIntervalIterable` without thinking about query logs. I would recommend to rename the parameters to `start` and `end`.